_includes/footer.html

```diff
@@ -1,77 +1,22 @@
-{% assign anchor = site.data.usa_anchor.anchor_data[0] %}
+{% assign identifier = site.data.usa_identifier %}
```

Comment:
[nitpick] since we're only using `identifier` in one place now, i think we can just use `site.data.usa_identifier.site_email` but also this isn't a huge deal so you can decide

Reply:
Mmm, that's a fair point. I'm tempted to leave in in case we do need to reference more identifier data in the future but, is that a bad practice?

Reply:
it's real tickytacky, so i think it's fine!